### PR TITLE
Implemented typography component

### PR DESCRIPTION
**Pull request by Manbeartrain**
Simulator Screen Shot - iPhone 14 Pro - 2022-10-05 at 20 31 51
Simulator Screen Shot - iPhone 14 Pro - 2022-10-05 at 20 32 04
Simulator Screen Shot - iPhone 14 Pro - 2022-10-05 at 20 32 11

### CODE REVIEW

File: src/components/Typography/Typography.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { StyleSheet, Text } from "react-native";
+
+import { styled } from "nativewind";
+
+import { TextType } from "./Typography.types";
+
+const Typography: React.FC<TextType> = ({ text, textColor, type }) => {
```
Comment (daboigbae) [inline src/components/Typography/Typography.tsx:9]: Let's make this Text instead of typography

File: src/components/Typography/Typography.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { StyleSheet, Text } from "react-native";
```
Comment (daboigbae) [inline src/components/Typography/Typography.tsx:3]: you can import this text using

Text as RnText 


That way whenever we see text, we know that we have to replace with one of ours

File: src/components/Typography/Typography.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+	const generatedStyle = generateTextStyle(textColor?.[0], type);
+
+	return <Text style={generatedStyle}>{text}</Text>;
+};
+
+export default styled(Typography, {
+	props: {
+		textColor: true,
+		type: true
```
Comment (daboigbae) [inline src/components/Typography/Typography.tsx:31]: should type be true here? it's a string from the typescript right? So maybe it doesn't need to be here

**Review by daboigbae — CHANGES_REQUESTED** [review]

**Comment by Manbeartrain** [comment]
[feature/text](https://trello.com/c/Ib6Qxlj0/16-feature-text)

**Review by daboigbae — APPROVED** [review]

**Pull request merged**